packages/salesforcedx-lightning-lsp-common/src/baseContext.ts

```diff
@@ -119,25 +121,28 @@ export const getModulesDirs = (
   workspaceRoots: string[],
   fileSystemProvider: IFileSystemProvider,
   getSfdxProjectConfig: () => SfdxProjectConfig
-): string[] => {
-  const modulesDirs: string[] = [];
+): NormalizedPath[] => {
+  // Normalize workspaceRoots at the start to ensure consistent path format
+  // This ensures all path operations use normalized paths
+  const normalizedWorkspaceRoots = workspaceRoots.map(root => utils.normalizePath(root));
+  const modulesDirs: NormalizedPath[] = [];
   switch (workspaceType) {
     case 'SFDX':
       const { packageDirectories } = getSfdxProjectConfig();
       for (const pkg of packageDirectories) {
         // Check both new SFDX structure (main/default) and old structure (meta)
-        const newPkgDir = path.join(workspaceRoots[0], pkg.path, 'main', 'default');
-        const oldPkgDir = path.join(workspaceRoots[0], pkg.path, 'meta');
+        const newPkgDir = path.join(normalizedWorkspaceRoots[0], pkg.path, 'main', 'default');
```

Comment:
that's kind of our default path, but there's never been a requirement to use `main/default` and some people don't.

Are we really just looking for "is there a dir named `lwc` in each packageDir" ?  If so, this is a clunky way to find it (doing a recursive readdir of them or using a fs.glob pattern) would be smarter.

Reply:
and then maybe we don't care about meta or main/default